docs/.env.example

```
@@ -77,6 +88,13 @@ OLLAMA_URL=http://localhost:11434
 # Options: DEBUG, INFO, WARNING, ERROR, CRITICAL
 LOG_LEVEL=INFO
 
+# Firecrawl Daemon Runtime
+# Poll interval (seconds) for queued jobs, retry backoff, and pagination safety cap
+FIRECRAWL_DAEMON_POLL_INTERVAL=5.0
+FIRECRAWL_DAEMON_RETRY_DELAY=60.0
+FIRECRAWL_DAEMON_MAX_RETRIES=3
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FIRECRAWL_DAEMON_MAX_RETRIES key should go before the FIRECRAWL_DAEMON_POLL_INTERVAL key

(UnorderedKey)